tmt/convert.py

```diff
@@ -170,6 +173,20 @@ def write_markdown(path, content):
         raise ConvertError(f"Unable to write '{path}'.")
 
 
+def add_bug(bug, data):
+    """ Add relevant bug into data under the 'link' key """
+    new_link = dict(verifies=f"{BUGZILLA_URL}{bug}")
+    try:
+        # Make sure there are no duplicates
+        for link in data['link']:
```

Comment:
Can it be  `new_link not in data['link']` instead?

Reply:
Good point, makes sense.

Reply:
Fixed in db246f7.